site/content/docs/5.2/customize/color.md

```diff
@@ -6,15 +6,373 @@ group: customize
 toc: true
 ---
 
-## Theme colors
+## Colors
+
+{{< added-in "5.3.0" >}}
+
+Bootstrap's color palette has continued to expand and become more nuanced in v5.3.0. We've added new variables for `secondary` and `tertiary` text and background colors, plus `{color}-bg-subtle`, `{color}-border-subtle`, and `{color}-text` for our theme colors. These new colors are available through Sass and CSS variables (but not our color maps or utility classes) with the express goal of making it easier to customize across multiple colors modes like light and dark. These new variables are globally set on `:root` and are adapted for our new dark color mode while our original theme colors remain unchanged.
+
+Colors ending in `-rgb` provide the `red, green, blue` values for use in `rgb()` and `rgba()` color modes. For example, `rgba(var(--bs-secondary-bg-rgb), .5)`.
+
+{{< callout warning>}}
+**Heads up!** There's some potential confusion with our new secondary and tertiary colors, and our existing secondary theme color, as well as our light and dark theme colors. Expect this to be ironed out in v6.
+{{< /callout >}}
+
+<table class="table table-swatches">
+  <thead>
+    <tr>
+      <th style="width: 340px;">Description</th>
+      <th style="width: 200px;" class="ps-0">Swatch</th>
+      <th>Variables</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td rowspan="2">
+        {{< markdown >}}**Body —** Default foreground (color) and background, including components.{{< /markdown >}}
+      </td>
+      <td class="ps-0">
+        <div class="p-3 rounded-2" style="background-color: var(--bs-body-color);">&nbsp;</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-body-color`<br>`--bs-body-color-rgb`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <div class="p-3 rounded-2 border" style="background-color: var(--bs-body-bg);">&nbsp;</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-body-bg`<br>`--bs-body-bg-rgb`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td rowspan="2">
+        {{< markdown >}}**Secondary —** For disabled component states, dividers, and lighter text.{{< /markdown >}}
+      </td>
+      <td class="ps-0">
+        <div class="p-3 rounded-2" style="background-color: var(--bs-secondary-color);">&nbsp;</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-secondary-color`<br>`--bs-secondary-color-rgb`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <div class="p-3 rounded-2 border" style="background-color: var(--bs-secondary-bg);">&nbsp;</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-secondary-bg`<br>`--bs-secondary-bg-rgb`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td rowspan="2">
+        {{< markdown >}}**Tertiary —** For component hover colors, accents, wells, and text.{{< /markdown >}}
```

Comment:
When I'm reading this color palette, I can understand that I could use `--bs-secondary-color` as a foreground color on `--bs-secondary-bg` as a background color (same thing for body and tertiary). But that it doesn't provide enough contrast for the tertiary for example. Am I understanding it correctly or not at all?

Reply:
The intent actually _isn't_ that these are used together, but at least for secondary, you could and you'd have a great contrast ratio. I think for the implied usage of a piece of "tertiary content" though, it'd be something that's not critical to the page and maybe safe to leave at a lower contrast? I'm forgetting the a11y term here that'd describe what I mean, but hopefully that helps.

In the mean time I'll clarify the usage a bit more.